packages/dcos-integration-test/extra/conftest.py

```diff
@@ -41,6 +42,33 @@ def pytest_collection_modifyitems(session, config, items):
     items[:] = new_items + last_items
 
 
+def pytest_addoption(parser):
+    parser.addoption('--resiliency', action='store_true')
+
+
+def pytest_runtest_setup(item):
+    if item.get_marker('resiliency'):
+        if not item.config.getoption('--resiliency'):
+            pytest.skip('Test requires --resiliency option')
+
+
+@pytest.yield_fixture
+def vip_apps(cluster):
+    vip1 = '6.6.6.1:6661'
+    test_app1, _ = cluster.get_test_app()
+    test_app1['portDefinitions'][0]['labels'] = {
+        'VIP_0': vip1}
+    test_app2, _ = cluster.get_test_app()
+    test_app2['portDefinitions'][0]['labels'] = {
+        'VIP_0': 'foobarbaz:5432'}
+    vip2 = 'foobarbaz.marathon.l4lb.thisdcos.directory:5432'
+    cluster.deploy_marathon_app(test_app1)
```

Comment:
I'll can this with:
with marathon_deploy_and_cleanup:
    with marathon_deploy_and_cleanup:
        yield

no different to me.